style.css

```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+.navbar {
+  display: -webkit-box;
+  /* stylelint-disable-next-line */
```

Comment:
- [ ] I suggest that you do not disable the linters, please remove this line and fix the linter instead. It is the good practice 💯